waku/node/waku_node.nim

```diff
@@ -346,6 +358,24 @@ proc selectRandomPeers*(peers: seq[PeerId], numRandomPeers: int): seq[PeerId] =
   shuffle(randomPeers)
   return randomPeers[0 ..< min(len(randomPeers), numRandomPeers)]
 
+proc mountRendezvousClient*(node: WakuNode, clusterId: uint16) {.async: (raises: []).} =
+  info "mounting rendezvous discovery protocol"
+
+  node.wakuRendezvous = WakuRendezVous.new(
```

Comment:
Mentioned below in another comment, but since we don't actually mount anything on the switch as a client, why not have a separate `WakuRendezvousClient` type?